scripts/flake8-diff.sh

```diff
@@ -5,4 +5,4 @@
 # E226 missing whitespace around arithmetic operator
 # F401 '._init_path' imported but unused
 # W504 line break after binary operator
-flake8 --diff --filename=*.py --ignore=E226,F401,W504 --max-line-length=88 --statistics
+flake8 --diff --filename=*.py --ignore=E226,E203,W503,F401,W504 --max-line-length=88 --statistics
```

Comment:
Please keep lists like this sorted in order to more easily spot duplicate or missing entries.  When the list is short then things are easy but issues will arise when lists get long.  The same best practice holds true for `requirements.txt`, and other lists.

I suggest removed `W504` because it is the opposite of `W503` which psf/black has chosen to enforce.
```suggestion
flake8 --diff --filename=*.py --ignore=E203,E226,F401,W503 --max-line-length=88 --statistics
```